packages/auth/src/authClients/nhost.ts

```diff
@@ -77,7 +77,7 @@ export const nhost = (client: Nhost): AuthClient => {
       return await client.auth.register({
         email,
         password,
-        registrationOptions: { userData: { display_name: email } },
+        options: { userData: { display_name: email } },
```

Comment:
@Tobbe @nunopato I think I may have upgraded a package or something that changed this. I checked NHosts library implementation and it seems to match.

https://github.com/nhost/nhost-js-sdk/blob/master/src/Auth.ts#L104